### PR TITLE
Makes shuttle load in transit level [TESTED]

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
Requires #44655 first.  Instead of using the centcomm shuttle loading landmark this reserves a block in the transit level to load it.

## Why It's Good For The Game
Less reliance of mapped in objects for core game functionality.

### CODE REVIEW

File: code/controllers/subsystem/shuttle.dm [hunk 1]
```diff
@@ -621,3 +633,272 @@ SUBSYSTEM_DEF(shuttle)
...
+	. = preview_shuttle
+
+	// Shuttle state involves a mode and a timer based on world.time, so
+	// plugging the existing shuttles old values in works fine.
+	preview_shuttle.timer = timer
+	preview_shuttle.mode = mode
+
+	preview_shuttle.register()
+
+	// TODO indicate to the user that success happened, rather than just
```
Comment (optimumtact) [inline code/controllers/subsystem/shuttle.dm]: hello

File: code/controllers/subsystem/shuttle.dm [hunk 2]
```diff
@@ -621,3 +633,272 @@ SUBSYSTEM_DEF(shuttle)
...
+			if(S)
+				. = TRUE
+				unload_preview()
+				load_template(S)
+				if(preview_shuttle)
+					preview_template = S
+					user.forceMove(get_turf(preview_shuttle))
+		if("load")
+			if(existing_shuttle == backup_shuttle)
+				// TODO make the load button disabled
```
Comment (optimumtact) [inline code/controllers/subsystem/shuttle.dm]: hellooooo

File: code/controllers/subsystem/shuttle.dm (hunk 1 above)
Comment (spookydonut) [reply code/controllers/subsystem/shuttle.dm]: This was in the other pr that just got merged lol

**Review by ninjanomnom — APPROVED** [review]
This commit list is why I squash merge 100% of the time

**Pull request merged**